.github/workflows/continuous_integration.yml

```diff
@@ -1,42 +1,191 @@
+    name: Actions Unit Tests
+    runs-on: ubuntu-latest
+    env:
+      GDRIVE_CREDENTIALS: ${{ secrets.GDRIVE_CREDENTIALS }}
+      MAILCHIMP_LIST: ${{ secrets.MAILCHIMP_LIST }}
+      MAILCHIMP_API_KEY: ${{ secrets.MAILCHIMP_API_KEY }}
+      ALGOLIA_APP_ID: ${{ secrets.ALGOLIA_APP_ID }}
+      ALGOLIA_SEARCH_KEY: ${{ secrets.ALGOLIA_SEARCH_KEY }}
+      ALGOLIA_DOCS_INDEX: ${{ secrets.ALGOLIA_DOCS_INDEX }}
+      RASA_X_HOST: ${{ secrets.RASA_X_DOMAIN }}
+      RASA_X_PASSWORD: ${{ secrets.RASA_X_PASSWORD }}
+      RASA_X_USERNAME: ${{ secrets.RASA_X_USERNAME }}
+      RASA_X_HOST_SCHEMA: ${{ secrets.RASA_X_HOST_SCHEMA }}
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        make install-dev
+    - name: Unit Tests
+      working-directory: ${{ github.workspace }}
+      run: |
+        pytest . -v
+  data-validation:
+    name: Data Validation
+    runs-on: ubuntu-latest
+    steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.8.0
+      with:
+        access_token: ${{ github.token }}
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade "pip<20"
+        pip install -r requirements-dev.txt
+    - name: Rasa Data Validation
+      working-directory: ${{ github.workspace }}
+      run: |
+        rasa data validate --debug
+  training-testing:
+    name: Testing Stories
+    runs-on: ubuntu-latest
+    needs: [data-validation]
+    steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.8.0
+      with:
+        access_token: ${{ github.token }}
+    - uses: actions/checkout@v1
+    - id: files
+      uses: jitterbit/get-changed-files@v1
+    - name: set_training
+      if: |
+          contains(  steps.files.outputs.all, 'data/' ) 
+          || contains(  steps.files.outputs.all, 'config.yml' ) 
+          || contains(  steps.files.outputs.all, 'domain.yml' )
+      run: echo "RUN_TRAINING=true"  >> $GITHUB_ENV
+    - name: Set up Python 3.7
+      if: env.RUN_TRAINING == 'true'
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      if: env.RUN_TRAINING == 'true'
+      run: |
+          make install-dev
+    - name: Cross-validate NLU model
+      id: cvnlu
+      if: |
+          ( 
+            ( 
+              contains( github.event.pull_request.labels.*.name, 'nlu_testing_required' ) 
+              && contains(  steps.files.outputs.all, 'data/nlu/' ) 
+            )
+            || contains(  steps.files.outputs.all, 'config.yml' ) 
+          )
+      run: |  
+          rasa test nlu -f 3 --cross-validation
+          python .github/workflows/format_results.py
```

Comment:
It seems to be ignoring this condition now, it ran cv although there was no label 🤔